content/chat/rooms/messages.textile

```diff
@@ -184,12 +184,18 @@ room.messages.send(text = "hello")
 
 h2(#update). Update a message
 
-<aside data-type='new'>
-<p>Message updates were added in version 0.3. They are only available in the JavaScript and React SDKs.</p>
-</aside>
+blang[javascript,react].
+  <aside data-type='new'>
+  <p>Message updates were added in version 0.3.</p>
+  </aside>
 
-blang[javascript].
-  Use the "@messages.update()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat-js.Messages.html#update method to update a message in a chat room. All users that are "subscribed":#subscribe to messages on that room will receive the update:
+blang[swift,kotlin].
+  <aside data-type='new'>
+  <p>Message updates were added in version 0.2.0.</p>
+  </aside>
```

Comment:
Minor, but you could just make this one note to drop some of the blangs (and do the same for deletes); `Message updates were added in version 0.3 for the JavaScript and React SDKs, and version 0.2 for the Kotlin and Swift SDKs.`

Reply:
Fixed